indico/modules/events/sessions/models/blocks.py

```diff
@@ -14,6 +14,7 @@
 from indico.core.db import db
 from indico.core.db.sqlalchemy.locations import LocationMixin
 from indico.core.db.sqlalchemy.util.models import auto_table_args
+from indico.modules.events.timetable import TimetableEntry
```

Comment:
Everywhere else we use the full import path: `from indico.modules.events.timetable.models.entries import TimetableEntry`. It's just coincidence that we import it in `__init__.py` as well